GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2231,7 +2231,18 @@ private void SetSplitterPositions()
             // Account for this shift. This is a workaround at best in the same way as for FormCommit.
             if (!RevisionsSplitContainer.Panel2Collapsed && RevisionsSplitContainer.FixedPanel == FixedPanel.Panel2)
             {
-                RevisionsSplitContainer.SplitterDistance -= 4;
+                try
+                {
+                    int adjustedDistance = RevisionsSplitContainer.SplitterDistance - 4;
+                    if (adjustedDistance > 0)
```

Comment:
❔
```suggestion
                    if (RevisionsSplitContainer.Panel1MinSize <= adjustedDistance && adjustedDistance <= RevisionsSplitContainer.Width - RevisionsSplitContainer.Panel2MinSize
```

Reply:
Yes, probably better according to the exception

Reply:
In fact, I didn't like to re-implement all the logic in the setter. I find it difficult to read/maintain.
I misread the setter logic and I thinking about it again, I think that the try/catch only is good enough because the if don't add value here... I will remove it.